fontforge/CMakeLists.txt

```diff
@@ -227,7 +227,7 @@ if(BUILD_SHARED_LIBS)
     # Both fontforge.exe and *.pyd Python modules look for libfontforge.dll in
     # their directory, so we need to produce a second copy.
     add_custom_command(TARGET fontforge POST_BUILD
-      COMMAND cp $<TARGET_FILE:fontforge> ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}
+            COMMAND ${CMAKE_COMMAND} -E copy $<TARGET_FILE:fontforge> ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}
```

Comment:
I'm generally ok with this change, but does it fix a specific bug or just improves style?

Reply:
QOL change.
It's for win compilation. cp not supported in win. While "Cmake copy" is cross platform.